.changeset/clever-insects-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"blitz": patch
+---
+
+Use process.kill("SIGINT") instead of process.kill("SIGABRT") on startCustomServer() for better compatibility with operative systems
```

Comment:
```suggestion
Use `SIGINT` signal instead of `SIGABRT` to stop the process, when using custom server for better compatibility with operative systems
```